phpBB/adm/style/acp_search_index.html

```diff
@@ -30,7 +30,7 @@ <h1>{{ lang('ACP_SEARCH_INDEX') }}</h1>
 
 		{{ backend.S_HIDDEN_FIELDS }}
 
-		<legend>{{ lang('INDEX_STATS') }}{{ lang('COLON') }} {{ backend.NAME }} {% if backend.S_ACTIVE %}({{ lang('ACTIVE') }}) {% endif %}</legend>
+		<legend>{{ lang('INDEX_STATS') ~ lang('COLON') }} {{ backend.NAME }} {% if backend.S_ACTIVE %}({{ lang('ACTIVE') }}) {% endif %}</legend>
```

Comment:
Not sure why those parenthesis are hardcoded?
`({{ lang('ACTIVE') }})`

The space here seems like its not necessary?
` {% endif %}</legend>`

Reply:
If there was no parenthesis it would end up producing `Backend name Active`.

Reply:
I know but IMHO those parenthesis should go in the language key string directly or use html entities. `&rpar;` and `&lpar;`

Or if `{{ lang('ACTIVE') }}` is in use elsewhere then change it to `{{ lang('SEARCH_ACTIVE') }}` and put them there.

Reply:
I think that's beyond the scope of this PR. The proper solution would be to have a language key with just the parenthesis, i.e. `'TEXT_IN_PARENTHESIS' => '(%s)'` and then use that together with the needed language key where appropriate instead of creating a basically duplicated language entry for every instance of a needed text in parenthesis.

Reply:
That would be the best solution indeed. 
Just those hard-coded parenthesis are really bad coding now a days.

Reply:
I have created a ticker for this https://tracker.phpbb.com/browse/PHPBB3-16750